### PR TITLE
Trim diffs (#31)

### PR DESCRIPTION
**Pull request by yairchu**
Note that I left an arbitrary 3 lines of context around diff..

### CODE REVIEW

**Review by Peaker — APPROVED** [review]
Maybe "3" should be a parameter for how many context lines?

**Comment by yairchu** [comment]
> Maybe "3" should be a parameter for how many context lines?

Done!

File: src/Opts.hs [hunk 1]
```diff
@@ -61,6 +62,10 @@ parser =
                 ( O.strOption
                     ( O.long "merge-file" <> O.short 'f' <> O.help "Merge a specific file")
                 )
+            <*> O.option O.auto
+                (O.long "context" <> O.metavar "LINECOUNT" <> O.showDefault <> O.value 3
+                    <> O.help "Number of context lines around dumped diffs"
```
Comment (Peaker) [inline src/Opts.hs:67]: Note that in `git diff` and `diff`, this is also called `-U`

Comment (yairchu) [reply src/Opts.hs:67]: Ah cool, added it now

**Pull request merged**